src/main/resources/com/google/api/codegen/java/standalone_sample.snip

```diff
@@ -41,42 +41,278 @@
+@private generateSample(apiMethod, sample)
+  @switch sample.callingForm
   @case "Request"
-    {@syncMethodSampleCode(apiMethod, initCode)}
+    {@syncMethodSampleCode(apiMethod, sample)}
+    {@processResponse(sample)}
   @case "RequestPaged"
-    {@pagedIterableMethodSampleCode(apiMethod, initCode)}
+    {@pagedIterableMethodSampleCode(apiMethod, sample)}
   @case "Flattened"
-    {@syncMethodSampleCode(apiMethod, initCode)}
+    {@syncMethodSampleCode(apiMethod, sample)}
+    {@processResponse(sample)}
   @case "FlattenedPaged"
-    {@pagedCallableMethodSampleCode(apiMethod, initCode)}      
+    {@pagedCallableMethodSampleCode(apiMethod, sample)}      
   @case "Callable"
-    {@callableMethodSampleCode(apiMethod, initCode)}
+    {@callableMethodSampleCode(apiMethod, sample)}
+    {@processResponse(sample)}
   @case "CallableList"
-    {@unpagedListCallableMethodSampleCode(apiMethod, initCode)}
+    {@unpagedListCallableMethodSampleCode(apiMethod, sample)}
   @case "CallablePaged"
-    {@pagedCallableMethodSampleCode(apiMethod, initCode)}
+    {@pagedCallableMethodSampleCode(apiMethod, sample)}
   @case "CallableStreamingBidi"
-    {@bidiStreamingCallableSampleCode(apiMethod, initCode)}      
+    {@bidiStreamingCallableSampleCode(apiMethod, sample)} 
   @case "CallableStreamingClient"
-    {@clientStreamingCallableSampleCode(apiMethod, initCode)}
+    {@clientStreamingCallableSampleCode(apiMethod, sample)}
   @case "CallableStreamingServer"
-    {@serverStreamingCallableSampleCode(apiMethod, initCode)}
+    {@serverStreamingCallableSampleCode(apiMethod, sample)}
   @case "LongRunningCallable"
-    {@asyncOperationCallableMethodSampleCode(apiMethod, initCode)}
+    {@asyncOperationCallableMethodSampleCode(apiMethod, sample)}
+    {@processResponse(sample)}
   @case "LongRunningFlattenedAsync"
-    {@asyncOperationMethodSampleCode(apiMethod, initCode)}
+    {@asyncOperationMethodSampleCode(apiMethod, sample)}
+    {@processResponse(sample)}
   @case "LongRunningRequestAsync"
-    {@asyncOperationMethodSampleCode(apiMethod, initCode)}
+    {@asyncOperationMethodSampleCode(apiMethod, sample)}
+    {@processResponse(sample)}
   @default
-    $unhandledCallingForm: {@callingForm} in sample "{@apiMethod.getClass.getSimpleName}"$
+    $unhandledCallingForm: {@sample.callingForm} in sample "{@apiMethod.getClass.getSimpleName}"$
   @end
 @end
 
+# The snippets are copied and modified from method_sample.snip since standalone
+# samples need to deal with output.
+@snippet pagedIterableMethodSampleCode(apiMethod, sample)
+  {@initCode(sample.initCode)}
+  for ({@apiMethod.listMethod.resourceTypeName} element : {@sampleSyncMethodCall(apiMethod)}.{@apiMethod.listMethod.iterateMethodName}()) {
+    {@processResponse(sample)}
+  }
+@end
+
+@snippet pagedCallableMethodSampleCode(apiMethod, sample)
+  {@initCode(sample.initCode)}
+  ApiFuture<{@apiMethod.responseTypeName}> future = {@sampleFutureMethodCall(apiMethod)};
+  // Do something
```

Comment:
Let's remove all these "Do something"s in favor of the response handling.

Reply:
I think these `//Do something`s are to show the `future.get()` does not have to be called immediately after a `future` is created?